dev/com.ibm.ws.security.kerberos.java8/src/com/ibm/ws/security/kerberos/internal/KerberosExtServiceImpl.java

```diff
@@ -145,4 +138,26 @@ public boolean isS4U2selfEnable() {
     public boolean isS4U2proxyEnable() {
         return (kerberosExtConfig != null ? kerberosExtConfig.isS4U2proxyEnable() : false);
     }
+
+    /**
+     * Register the webcontainer's default delegation provider.
+     *
+     * @param cc
+     */
+    private void registerKrb5HelperJdk(ComponentContext cc) {
+        Dictionary<String, Object> props = new Hashtable<String, Object>();
+        if (Krb5LoginModuleWrapper.IBM_KRB5_LOGIN_MODULE_AVAILABLE) {
+            krb5HelperJdk = new IBMKrb5Helper();
```

Comment:
Do you have any concern that you are newing up these directly and have them in the imports so that they could end up trying to load IBM Java 8 kerberos on a non IBM JDK and not find them and get NoClassDefFoundErrors?  Consequently should this use Class.forName instead and calling newInstance()?